prompt_guardian/server.py

```diff
@@ -104,52 +104,57 @@ async def add_url(url_add_request: URLAddRequest, request: Request):
     return {"status": "URL added to the list"}
 
 
-@prompt_guardian_app.post("/check-prompt")  # Note the change to a POST request
-async def check_url(prompt_check_request: PromptCheckRequest, request: Request):
-    prompt = prompt_check_request.text
-    urls = prompt_check_request.extractedUrls
```

Comment:
This seems to be outdated and not being used anywhere, hence removing it from the source code